lib/active_shipping/carriers/ups.rb

```diff
@@ -1201,5 +1201,9 @@ def package_level_delivery_confirmation?(origin, destination)
       origin.country_code == destination.country_code ||
       [['US','PR'], ['PR','US']].include?([origin,destination].map(&:country_code))
     end
+
+    def patch_country_code(country_code)
```

Comment:
I like this, makes countries easier to ~~patch~~ map in the future should we run into this again. 

Naming was tough, went back and forth with @jonathankwok on that one but I do think `map` describes the action best.